### ISSUE TITLE
Intel 8260 appears as Ethernet but can't connect.

### ISSUE THREAD

**Issue by hillfieldhonest** [zxystd/itlwm#102]
Hi, 首先很感谢 这个IntelWIfi Project！我最近买到 Intel Compute Stick(m3) (which comes with Intel Wifi 8260) 本来USB端口也不多，所以我也一直找个办法如何用内置的IntelWifi.  and finally I found this awesome project! hopefully I can get mine work too.  

My Env:
HW: Compute Stick STK2M3W64CC
Installation: OpenCore 0.5.8 refering to following guide (for Skylake Desktop) 
https://dortania.github.io/OpenCore-Desktop-Guide/
kext: I used the binary one from here.
https://www.tonymacx86.com/threads/success-working-intel-wifi-drivers-for-7265ac-on-catalina.292207/page-49#post-2129921

Steps:
loaded itlwm kext manually. after editting info.plist (password only includes numbers)
cp the kext to tmp. then
sudo chown R root:wheel /tmp/itlwm.kext
sudo kextutil -v 6 /tmp/itlwm.kext 

Result:
kext loaded and it will show as Ethernet but it doesn't connect to internet.

note:
log file and some screenshots attached. let me know if more files are needed.
[log.log](https://github.com/zxystd/itlwm/files/4709459/log.log)
![PCIe_Hackintool](https://user-images.githubusercontent.com/49436513/83377458-7f46cd80-a410-11ea-917b-de86256b75f6.png)
![Peripherials_Hackintool](https://user-images.githubusercontent.com/49436513/83377462-8241be00-a410-11ea-8d84-1dccde77286a.png)

**Comment by blickers** [comment]
Be careful when editing info.plist: password comes before ssid. Check if u've filled them correctly.

**Comment by hillfieldhonest** [comment]
thank you for the advise. I double checked my info.plist just in case.
I believe I wrote it in correct way. are there anywhere else to check?

![info plist_img](https://user-images.githubusercontent.com/49436513/83480931-8258c100-a4d7-11ea-9a07-cc6f3736d326.png)

**Comment by zxystd** [comment]
![image](https://user-images.githubusercontent.com/16227538/83481372-2d657c80-a4d0-11ea-93a7-a11c6dc4e591.png)
Only 'MakotoWifi2' in the scan result, no 'MakotoWifi', check it?

**Comment by hillfieldhonest** [comment]
thank you for checking. and sorry for the confusion.
I actually had changed the ssid and retested it.
here is the log after the change. I can find the correct ssid "MakotoWifi" here...
[log_2.log](https://github.com/zxystd/itlwm/files/4714877/log_2.log)

**Comment by zxystd** [comment]
itlwm: sending auth to 14:dd:a9:a3:2a:9d on channel 6 mode 11g
itlwm: authentication timed out for 14:dd:a9:a3:2a:9d

Router didn't accept the auth, can you tell me the WiFi configuration? Now it can only connect 40mhz bandwidth, 11n mode WiFi.

**Comment by hillfieldhonest** [comment]
I'm using Wi-Fi tethering of Android (ZenFone 2 ZE551ML according to the spec, it supports a/ac/b/g/n. but there is no setting available to set which it use for the Wifi tethering feature.)

so, currently it tries to connect 40Mhz bandwidth 11n mode Wifi? it's good to know, I can try with other routers or phones..

**Comment by hillfieldhonest** [comment]
I tried connecting to the "MakotoWifi" to know what the connection type is: it shows like this:

at least it connect using 11n mode. however not sure about 40Mhz bandwidth or not though..

![onWin](https://user-images.githubusercontent.com/49436513/83488471-dc618280-a4e7-11ea-9b85-94feb70b1c2f.JPG)

**Comment by zxystd** [comment]
And you check password is correct? Maybe you can use another wifi to test...

**Comment by hillfieldhonest** [comment]
yes. I double checked the password.
I also tried another Androids' wifi tethering, and also Windows' mobile hotspot but in vain...
I will test again with another. and will update with the result. 

btw, eventually. 20Mhz 11n mode/or other mode will be supported?

**Comment by hillfieldhonest** [comment]
I tried with the last hope, i mean only one left wi-fi router. 

and It worked!!!! I changed the router setting to 802.11/a/n/ac mode (5Ghz) , and set bandwidth to auto (since there was no 40Mhz option.)  and now it works like a magic! 

I really appreciate your great help. 
I'm posting this using the Compute stick using this itlwm kext. I'm really moved. 
![Screen Shot 2020-06-02 at 23 18 36](https://user-images.githubusercontent.com/49436513/83532129-d213a880-a528-11ea-8bf6-e8543697a885.png)
![Screen Shot 2020-06-02 at 23 18 49](https://user-images.githubusercontent.com/49436513/83532138-d4760280-a528-11ea-99ce-6171773d4e11.png)
